app/build.gradle

```diff
@@ -537,6 +537,7 @@ dependencies {
     debugImplementation Deps.leakcanary
 
     implementation Deps.androidx_compose_ui
+    implementation Deps.androidx_compose_livedata
```

Comment:
This can probably be dropped in favor of using the simpler `coroutines.flow.StateFlow` api.